src/views/dashboard/DashboardLanding.js

```diff
@@ -68,7 +68,7 @@ class DashboardLanding extends Component {
             <Route path='/dashboard/inbox' render={(props) => <Inbox {...props} />} />
             <Route path='/dashboard/myweek' render={(props) => <MyWeek {...props} />} />
             <Route path='/dashboard/Admin' render={(props) => <Admin {...props} />} />
```

Comment:
@shawnnostrom all these renders does not pass any specific prop down, so these could be just something like ```component={Admin}```